src/App.jsx

```diff
@@ -1,41 +1,15 @@
 import { ThemeProvider } from '@mui/material/styles';
-import { useEffect, useState } from 'react';
+import { useState } from 'react';
 import { BrowserRouter, Route, Routes } from 'react-router-dom';
 
-import { readAll } from '/src/firebase';
 import SellHousePage from '/src/pages/SellHousePage/SellHousePage';
 import theme from '/src/theme/theme';
 
 import HomePage from './pages/HomePage/HomePage';
 
 function App() {
-  const [houses, setHouses] = useState(null);
-  const [error, setError] = useState(false);
-  const [loading, setLoading] = useState(true);
   const [LoggedIn, setLoggedIn] = useState(false);
```

Comment:
Actually this could also be removed because Tomek's auth context is already on `develop` but this could be done once you resolve conflicts.